examples/with-graphql-hooks/components/#app.js#

```diff
@@ -0,0 +1,44 @@
+export default function App({ children }) {
```

Comment:
This file seems to be accidentally committed 🤔

Reply:
Removed it.